app/inithttp.go

```diff
@@ -165,6 +166,13 @@ func (app *App) initHTTP(ctx context.Context) error {
 		UserStore:           app.UserStore,
 	})
 
+	slack := slack.NewHandler(slack.Config{
+		AlertStore:    app.AlertStore,
+		AlertLogStore: app.AlertLogStore,
+		UserStore:     *app.UserStore,
+		AuthHandler:   *app.AuthHandler,
+	})
```

Comment:
Slack shouldn't be accessing these stores, it should handle signature validation itself and pass info to the engine for responses where things can be authenticated.

Reply:
where in the engine would the logic be handled from the request?